package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "metal-metal-key",
+  "version": "0.0.0",
+  "description": "Adds a custom event that helps listen to key events in a cleaner way",
+  "license": "BSD",
+  "repository": "fernandosouza/metal-metal-key",
+  "engines": {
+    "node": ">=0.12.0",
+    "npm": ">=3.0.0"
+  },
+  "main": "lib/all/metalKey.js",
+  "jsnext:main": "src/MetalKey.js",
+  "files": [
+    "lib",
+    "src",
+    "test"
+  ],
+  "scripts": {
+    "build": "gulp build",
+    "compile": "babel --presets metal -d lib/ src/",
+    "prepublish": "npm run compile",
+    "start": "http-server . -p 4000",
```

Comment:
Is this server going to be used by developers for anything? If not, it's best to remove it.